pyproject.toml

```diff
@@ -40,7 +40,7 @@ test=[
     "pytest-cov",
 ]
 dev=[
-    "torch_frame[test]",
+    "pytorch_frame[test]",
```

Comment:
Just for the note,

```
[tool.flit.module]
name="torch_frame"
```

is fine as we're supposed to fill it with import name but not with package name on PyPI.

Reply:
https://flit.pypa.io/en/stable/pyproject_toml.html#module-section